services/github/github-hacktoberfest.service.js

```diff
@@ -0,0 +1,194 @@
+  transformErrors,
+} = require('./github-helpers')
+
+const documentation = `
+  <p>
+    This badge is designed for projects hosted on GitHub which are
+    participating in
+    <a href="https://hacktoberfest.digitalocean.com">Hacktoberfest</a>,
+    an initiative to encourage participating in open-source projects. The
+    badge can be added to the project readme to encourage potential
+    contributors to review the suggested issues and to celebrate the
+    contributions that have already landed.
+
+    The badge displays three pieces of information:
+    <ul>
+      <li>
+        The number of suggested issues. By default this will count open
+        issues with the <strong>hacktoberfest</strong> label, however you
+        can pick a different label (e.g.
+        <code>?suggestion_label=good%20first%20issue</code>).
+      </li>
+      <li>
+        The number of pull requests opened in October. This excludes any
+        PR with the <strong>invalid</strong> label.
+      </li>
+      <li>The number of days left of October.</li>
+    </ul>
+
+  </p>
+
+  ${githubDocumentation}
+`
+
+const schema = Joi.object({
+  data: Joi.object({
+    repository: Joi.object({
+      issues: Joi.object({
+        totalCount: nonNegativeInteger,
+      }).required(),
+    }).required(),
+    search: Joi.object({
+      issueCount: nonNegativeInteger,
+    }).required(),
+  }).required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  suggestion_label: Joi.string(),
+}).required()
+
+module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Service {
+  static get category() {
+    return 'issue-tracking'
+  }
+
+  static get route() {
+    return {
+      base: 'github/hacktoberfest/2019',
+      pattern: ':user/:repo',
+      queryParamSchema,
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'GitHub Hacktoberfest combined status',
+        namedParams: {
+          user: 'snyk',
+          repo: 'snyk',
+        },
+        staticPreview: this.render({
+          suggestedIssueCount: 12,
+          contributionCount: 8,
+          daysLeft: 15,
+        }),
+        documentation,
+      },
+      {
+        title:
+          'GitHub Hacktoberfest combined status (suggestion label override)',
+        namedParams: {
+          user: 'tmrowco',
+          repo: 'tmrowapp-contrib',
+        },
+        queryParams: {
+          suggestion_label: 'help wanted',
+          contribution_label: undefined,
+        },
+        staticPreview: this.render({
+          suggestedIssueCount: 12,
+          contributionCount: 8,
+          daysLeft: 15,
+        }),
+        documentation,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'hacktoberfest',
+      color: 'orange',
+    }
+  }
+
+  static render({ suggestedIssueCount, contributionCount, daysLeft }) {
+    if (daysLeft === undefined) {
+      daysLeft = moment('2019-11-01').diff(moment(), 'days')
```

Comment:
I suppose the days remaining count will be based on the either the time zone in France or Canada where our badge servers are deployed. I think that's fine (trying to do time zone localization seems more trouble than it'd be worth), though perhaps it'd be worth a mention in the docs?

Reply:
Hacktoberfest runs from `30 SEP 2019 10:00:00 UTC` till `01 NOV 2019 12:00:00 UTC` to allow for all timezones, so using that might work best?

Reply:
Hmm I was thinking it'd be UTC, though I suppose you're right that it's going to be the time on the server.

I'd suggest we dodge the issue by picking a UTC end time slightly into 11/1 when we expect it's rolled over to 11/1 for most programmers in the world.

@MattIPv4 am I right that it ends at midnight in the contributor's local time on 10/31? Or is there a coordinated end time?

Reply:
Global cut-off will be `01 NOV 2019 12:00:00 UTC`.

Reply:
Noon UTC on 11/1, sounds good.